### ISSUE TITLE
[HELP] - Use with an Legacy App that have child window as an popup.

### ISSUE THREAD

**Issue by lovedota** [bahmutov/cypress-open-child-window#1]
Hi @bahmutov 
I have an old application that is using child window as an popup, for example I have a List on a website with a create button, when click, it opens a new window  to create new item, after item is successful created, it used **window.opener.refereshList**.
Is there any chance that I can use your approach to test it rather doing a tweak to get the link of the child window and redirect to it then redirect back to the list to see the List is refreshed.

Basically I tried your **cypress-open-child-window** and it works, But I can't use `cy.get` the element in the child window.
<img width="999" alt="Screen Shot 2019-12-26 at 06 31 54" src="https://user-images.githubusercontent.com/417914/71451493-96222d80-27a9-11ea-99b4-ae77832eb638.png">

**Comment by jakedowns** [comment]
would you be willing to post your code? what version of cypress are you using?

the openWindow helper works great for me on ` 4.9.0`

Here's a gist for how i'm using it including a method for returning back to the previous context via cy.toggleWindows() or cy.closeWindow()

https://gist.github.com/jakedowns/32b75add88736fcd99c7202a98354384
